src/models/index.js

```diff
@@ -34,5 +33,6 @@ Object.keys(db).forEach((modelName) => {
 
 db.sequelize = sequelize;
 db.Sequelize = Sequelize;
+db.sequelize.sync();
```

Comment:
why did you add line 36?. What were you trying to achieve here?. Please, shed more light on this.

Reply:
`db.sequelize.sync()` creates tables that do not exist. Therefore running `yarn dev-start` returns error because there are no tables. 
A better approach could be to call `yarn migrate` in the `dev-start` script in package.json thus:
```
{
  dev-start: yarn migrate && ...
}
```
and then delete the line 36 `db.sequelize.sync()`.

Please could you tell your opinion? What do I do?